README.md

```diff
@@ -1 +1,3 @@
-# Study-group
\ No newline at end of file
+# Study-group
+
+## Start program with command node hello.js
```

Comment:
Add small description what program does (prints the name 5x). Also, move node hello.js into new line like this: 

```
node hello.js
```
so it uses the code formatting and README will look much nicer. Otherwise looks great! 🎉